modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -542,3 +561,180 @@ class ApiClient(object):
                     raise ApiValueError(
                         'Authentication token must be in `query` or `header`'
                     )
+
+{{#hasHttpSignatureMethods}}
+    def get_http_signature_headers(self, resource_path, method, headers, body, query_params):
+        """
+        Create a cryptographic message signature for the HTTP request and add the signed headers.
+        
+        :param resource_path : resource path which is the api being called upon.
+        :param method: the HTTP request method.
+        :param headers: the request headers.
+        :param body: body passed in the http request.
+        :param query_params: query parameters used by the API.
+        :return: instance of digest object 
+        """
+        if method is None:
+            raise Exception("HTTP method must be set")
+        if resource_path is None:
+            raise Exception("Resource path must be set")
+        if body is None:
+            body = ''
+        else:
+            body = json.dumps(body)
+
+        target_host = urlparse(self.configuration.host).netloc
+        target_path = urlparse(self.configuration.host).path
+
+        request_target = method.lower() + " " + target_path + resource_path
+
+        if query_params:
+            raw_query = urlencode(query_params).replace('+', '%20')
+            request_target += "?" + raw_query
+
+        # Get current time and generate RFC 1123 (HTTP/1.1) date/time string.
+        from email.utils import formatdate
+        now = datetime.datetime.now()
+        stamp = mktime(now.timetuple())
+        cdate=formatdate(timeval=stamp, localtime=False, usegmt=True)
```

Comment:
Can you add spaces around the equal sign on this line and the next line after the variable assignment?
```
 cdate = blah
created = blah
```

Reply:
yes